src/jsonp-ploader.js

```diff
@@ -10,11 +10,12 @@ import loader from './loader';
 export default class pLoader extends loader {
   /**
    * redirect external stream handler function
+   * @param {Object} data - The json object that returns from the server.
    * @param {string} uri - the original uri
    * @returns {string} uri - the redirected URI
    * @static
    */
-  static redirectExternalStreamsHandler: Function = uri => uri;
+  static redirectExternalStreamsHandler: Function = (data, uri) => uri;
```

Comment:
i didn't find where this was changed

Reply:
it wasn't changed. it was incorrect from begin. see [here](https://github.com/kaltura/playkit-js/blob/master/src/utils/jsonp.js#L51).
in real playback we overriding this function as u can see [here](https://github.com/kaltura/playkit-js-hls/blob/master/src/hls-adapter.js#L138). so it impacts the test only